include/utility/dbus_utility.hpp

```diff
@@ -197,5 +197,39 @@ inline bool callPIM(types::ObjectMap&& objectMap)
     }
     return true;
 }
+
+/**
+ * @brief API to check D-Bus service status.
+ *
+ * Any failure in calling the method "NameHasOwner" implies that the service is
+ * not in a running state. Hence the API returns false in case of any exception
+ * as well.
+ *
+ * @param[in] i_serviceName - D-Bus service name whose status is to be checked.
+ * @return bool - True if the service is running false otherwise.
+ */
+inline bool checkServiceStatus(const std::string& i_serviceName)
+{
+    bool l_retVal = false;
+
+    try
+    {
+        auto l_bus = sdbusplus::bus::new_default();
+        auto l_method = l_bus.new_method_call(
+            "org.freedesktop.DBus", "/org/freedesktop/DBus",
+            "org.freedesktop.DBus", "NameHasOwner");
```

Comment:
why did you change from NameOwnerChanged to NameHasOwner?

Reply:
Did I change that?